src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -235,6 +235,16 @@ public String toString() {
                 getQualsString());
     }
 
+    public String getPileupString(final Character ref) {
```

Comment:
document what this method does, change `Character` to `char` and add a unit test for it